src/components/NavMenu.jsx

```diff
@@ -0,0 +1,59 @@
+import { useNavigate } from 'react-router-dom';
+
+function NavMenu() {
+
+  const token = localStorage.getItem('token')
+  const navigate = useNavigate()
+
+  const handleLogout = () => {
+    localStorage.removeItem('token');
+    navigate('/login');
+    // Optionally redirect to login page or handle other logout logic
+  }; 
+    
+  return (
+    <div>
+      <nav className="bg-white dark:bg-gray-900 fixed w-full z-20 top-0 start-0 border-b border-gray-200 dark:border-gray-600">
+        <div className="max-w-screen-xl flex flex-wrap items-center justify-between mx-auto p-4">
+          <span className="self-center whitespace-nowrap text-2xl font-semibold text-indigo-700 dark:text-gray">
+            ScreenShots
+          </span>
+          <div className="flex md:order-2 space-x-3 md:space-x-0 rtl:space-x-reverse">
+            {token ? (
+              <button 
+                onClick={handleLogout}
+                type="button" 
+                className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">
+                Logout
+              </button>
+              ) :
+              ( <button 
+              
+                  type="button" 
+                  className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">
+                  Login
+                </button>
+              )
+            }
+                     
+            <button data-collapse-toggle="navbar-sticky" type="button" className="inline-flex items-center p-2 w-10 h-10 justify-center text-sm text-gray-500 rounded-lg md:hidden hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600" aria-controls="navbar-sticky" aria-expanded="false">
+              <span className="sr-only">Open main menu</span>
```

Comment:
What does this button do? Could you add some screenshots of the nav-bar before and after log-in?

Reply:
The Navbar only appears when a user logs in. 

<img width="593" alt="Screenshot_3" src="https://github.com/user-attachments/assets/5dccccb5-282f-4b97-a6b6-3e047a44f7ff">

The ` <button data-collapse-toggle="navbar-sticky"`, let the user on small screens to see the linking options available in the Navbar. In this case and until now, we  have only **Home**.

The button marked with the red arrow is used in responsive applications. 
The Navbar looks like this on small screens. 


<img width="462" alt="Screenshot_5" src="https://github.com/user-attachments/assets/8f26cc85-90f5-4769-9f95-74d5fdf28a2c">

This tag `<span className="sr-only">Open main menu</span>` is for  screen reader users.